source/develop/release-management/features/network/manageiq_ovn.html.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Integrating OVN to ManageIQ
+category: feature
+authors: Alona Kaplan
+wiki_category: Feature
+wiki_title: Integrating OVN to ManageIQ
+wiki_revision_count: 1
+wiki_last_updated: 2017-07-09
+feature_name: Integrating OVN to ManageIQ
+feature_modules: manageiq_providers_ovirt - infra_manager, network_manager, manageiq-ui-classic - network related controllers
+feature_status: In Development
+---
+
+# Integrating OVN to ManageIQ
+
+## Summary
+Let the ManageIQ operator discover OVN as the network provider of oVirt and adding the ability to show, add, update and remove networks, subnets and ports directly from `ovirt-provider-ovn`.
```

Comment:
Maybe:
The feature will allow the MangeIQ .... and add the ability to show .....

Reply:
Changed to Dan's proposal - 'Let the ManageIQ operator discover OVN as the network provider of oVirt and adding the ability to show, add, update and remove networks, subnets and ports directly from ovirt-provider-ovn.'